config/config.go

```diff
@@ -93,9 +96,6 @@ type BaseConfig struct {
 	// so the app can decide if we should keep the connection or not
 	FilterPeers bool `mapstructure:"filter_peers"` // false
 
-	// What indexer to use for transactions
```

Comment:
I don't have much context into this PR but why are we removing TxIndex from BaseConfig?

Reply:
not removing. moving into its own section